sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -319,7 +319,7 @@ protected static void trackPage(String name, Map<String, String> properties) {
      * Additional validation rules apply depending on the configured secret.
      * <p>
      * For App Center, the name cannot be longer than 256 and is truncated otherwise.
-     * For One Collector, the name needs to match the <tt>[a-zA-Z0-9]((\.(?!(\.|$)))|[_a-zA-Z0-9]){3,99}</tt> regular expression.
+     * For One Collector, the name needs to match the <code>[a-zA-Z0-9]((\.(?!(\.|$)))|[_a-zA-Z0-9]){3,99}</code> regular expression.
```

Comment:
Im not sure - what are those changes?

Reply:
Javadoc errors
`error: tag not supported in the generated HTML version: tt`